.hound.yml

```diff
@@ -1,5 +1,5 @@
 fail_on_violations: true
 eslint:
- enabled: true
- config_file: .eslintrc.json
- ignore_file: .eslintignore
+  enabled: true
```

Comment:
@Ramadhan0 this file shouldn't be in your PR.

Reply:
Here, me and Premise we were switching between ```.eslintrc.json``` and ```.eslintrc.js``` in order to ensure how those files were working.

Reply:
Alright, let's undo the changes here since it's not related to your PR.